src/Configuration/Properties/PropertyCollection.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration\Properties;
+
+use ArrayObject;
+
```

Comment:
The phpStorm stubs for ArrayObject (and all other iterables) now ship with key-value generics (psalm template). We could define them here!